src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_x.h

```diff
@@ -253,15 +253,31 @@ namespace sirf {
 			return "ImagesReconstructor";
 		}
 
+		void set_dcm_prefix(const char* dcm_prefix)
+		{
+			dcm_prefix_ = dcm_prefix;
+		}
+		std::string dcm_prefix() const
+		{
+			return dcm_prefix_;
+		}
+		bool dcm_output() const
+		{
+			return dcm_prefix_.size() > 1;
```

Comment:
probably should be `>0` I guess, but better to check `!dcm_prefix.empty()`

Reply:
done